Common/src/config_structure.cpp

```diff
@@ -4286,6 +4290,14 @@ void CConfig::SetPostprocessing(unsigned short val_software, unsigned short val_
       SU2_MPI::Error("Different number of topology filter kernels and respective radii.", CURRENT_FUNCTION);
     }
   }
+  
+  /*--- If we are executing SU2_DOT in surface file mode, then
+   force the projected surface sensitivity file to be written. ---*/
+  
+  Wrt_Projected_Sensitivity = false;
+  if ((Kind_SU2 == SU2_DOT) && (Design_Variable[0] == SURFACE_FILE)) {
+    Wrt_Projected_Sensitivity = true;
```

Comment:
Usually by default, we have a file written out called surface_sens.dat (e.g. Tecplot ASCII) when calling SU2_DOT. Why is this routine needed? Is it also for keeping a certain format? From looking through the code, it is very similar to `SetTecplotASCII`.

Reply:
Yep, it is for the format. Basically, the visualization files contain other information like headers and the connectivity for the mesh. In the simple ASCII output here, it is simply the x,y,z, and sens values (unordered), which is smaller/easier to handle downstream. Also, depending on whether you write ParaView or Tecplot files for the surface sens, it will also be different and require special handling downstream. Basically, it would be good to have a fixed format for this if it will be relied upon in optimization loops.

Reply:
Ok.